docs/documentation/head/setup.md

```diff
@@ -42,3 +42,11 @@ Where MM is the major version, nn is the minor version and pp is the patch versi
 Versions for JDBC3 and lower can be found [here](https://jdbc.postgresql.org/download.html#others)
 This is a very brief outline of how to build the driver. Much more detailed information can be 
 found on the [github repo](https://github.com/pgjdbc/pgjdbc/blob/master/CONTRIBUTING.md)
+
+Even though the JDBC driver should be built with Gradle, for situations, where use of Gradle is not possible,
+e.g., when building pgjdbc for distributions, the pgjdbc gradle build provides a convenience
+source release artifact **-src.tar.gz** - a Maven based project.
+The Maven based project contains a minimal version of the JDBC driver with reduced feature and test set
+and is still validly buildable within the Maven build environment.
+Minimal version of the JDBC driver is created with **gradlew -d :postgresql:sourceDistribution -Prelease**
```

Comment:
I feel the need to apologize for being so pedantic. I really do appreciate these contributions. However  I think I'd be more comfortable with.
```
Using Maven and this artifact a complete JDBC driver can be built and used in production. 
```

I would also add that the resulting sourceDistribution will be found in `pgjdbc/build/distributions/`
and
in order to build the jar using maven once you have extracted the source files is

mvn package
or 
mvn -DskipTests package

Reply:
> I feel the need to apologize for being so pedantic. I really do appreciate these contributions. However, I think I'd be more comfortable with.

Well, I think it is better to clarify things right from the start. My knowledge of the JDBC driver is limited; sorry for generating more work for you.

> Using Maven and this artifact a complete JDBC driver can be built and used in production. 

And is it really true, that the JDBC driver build from this artifact would be complete? In all conversations (e.g., here [1]) about this topic, I read only about minimal JDBC driver with a reduced feature set, not a complete one. I also compared the source files of the original JDBC driver and this maven one, and they are not the same, some files are missing in the Maven-based artifact.

I am going to add the rest of the information you mention, thanks.

[1] https://github.com/pgjdbc/pgjdbc/issues/1440